dashboard/src/components/Catalog/Catalog.test.tsx

```diff
@@ -212,15 +213,17 @@ describe("filters by application repository", () => {
   });
 
   it("push filter for repo", () => {
-    const store = getStore({});
-    const wrapper = mountWrapper(store, <Catalog {...populatedProps} />);
+    const store = getStore({ repos: { repos: [{ metadata: { name: "foo" } } as IAppRepository] } });
+    const fetchRepos = jest.fn();
+    const wrapper = mountWrapper(store, <Catalog {...populatedProps} fetchRepos={fetchRepos} />);
     // The repo name is "foo"
     const input = wrapper.find("input").findWhere(i => i.prop("value") === "foo");
     input.simulate("change", { target: { value: "foo" } });
     // It should have pushed with the filter
     const historyAction = store
       .getActions()
       .find(action => action.type === "@@router/CALL_HISTORY_METHOD");
+    expect(fetchRepos).toHaveBeenCalledWith("kubeapps");
```

Comment:
So this tests the case where the namespace is kubeapps, but we could also update a separate test to ensure that
```
expect(fetchRepos).toHaveBeenCalledWith("otherNamespace", true);
```
?

Reply:
:heavy_plus_sign: